client/interfaces.go

```diff
@@ -314,6 +314,7 @@ type InstanceServer interface {
 	UpdateClusterMember(name string, member api.ClusterMemberPut, ETag string) (err error)
 	RenameClusterMember(name string, member api.ClusterMemberPost) (err error)
 	CreateClusterMember(member api.ClusterMembersPost) (op Operation, err error)
+	UpdateClusterCertificate(certs api.ClusterCertificatePut) (err error)
```

Comment:
All our `Update*` functions tag an `ETag` so this one should too (even though it's currently impossible to get that value given there's no matching GET).